prow/plugins/trigger/generic-comment.go

```diff
@@ -156,13 +157,16 @@ func handleGenericComment(c Client, trigger plugins.Trigger, gc github.GenericCo
 						"repo":    repo,
 					})
 					runID := run.ID
-					go func() {
-						if err := c.GitHubClient.TriggerFailedGitHubWorkflow(org, repo, runID); err != nil {
-							log.Errorf("attempt to trigger github run failed: %v", err)
-						} else {
-							log.Infof("successfully triggered action run")
+					if err := c.GitHubClient.TriggerFailedGitHubWorkflow(org, repo, runID); err != nil {
+						log.Errorf("attempt to trigger github run failed: %v", err)
+						if strings.Contains(err.Error(), "Unable to retry this workflow run because it was created over a month ago") {
+							resp := "The workflow triggered has expired. You can find more information about the same here <>"
+							c.Logger.Infof("Commenting \"%s\".", resp)
+							return c.GitHubClient.CreateComment(org, repo, number, plugins.FormatResponseRaw(gc.Body, gc.HTMLURL, gc.User.Login, resp))
```

Comment:
Thinking more about the comment here, ideally we would want our comment to include the name of each expired workflow, something like this:
```
The following workflows could not be re-triggered because they are now expired (created over a month ago):
- name1
- name2
...
```

It is possible to comment as and when this happens and then keep editing the comment, but instead of doing that and to keep things simple, let's just keep track of the names of expired workflows in a slice and keep appending to the slice. 

Once we are out of the loop, check if the `len` of that slice is `>0` and if it is, then post a comment with all the names of the expired workflows.